src/lib/server/request.c

```diff
@@ -38,18 +38,18 @@ fr_dict_autoload_t request_dict[] = {
 };
 
 fr_dict_attr_t const *request_attr_root;
-fr_dict_attr_t const *request_attr_request;
-fr_dict_attr_t const *request_attr_reply;
-fr_dict_attr_t const *request_attr_control;
-fr_dict_attr_t const *request_attr_state;
+fr_dict_attr_t const *fr_request_attr_request;
+fr_dict_attr_t const *fr_request_attr_reply;
+fr_dict_attr_t const *fr_request_attr_control;
+fr_dict_attr_t const *fr_request_attr_state;
```

Comment:
Not sure we want the `fr_` prefix, it's only usually used in libraries we might want to use in other applications, so it's not traditionally been added to things in the server library.

Reply:
I'll take that out - I misunderstood the reasoning behind the `HIDDEN` and thought it was to do with the naming.